### ISSUE TITLE
Create a structure from a CONTCAR file 

### ISSUE THREAD

**Issue by jan-janssen** [pyiron/pyiron_atomistics#570]
Yiting Zhang from Darmstadt submitted a question via the pyiron website:

> Hello, I would like to know if it is possible to create a structure with CONTCAR, such that it not only contains the positions of the atoms in the CONTCAR file, but also the velocities and Predictor-corrector coordinates.

@sudarsan-surendralal I guess we have https://pyiron.readthedocs.io/en/pyiron-0.2.17/apidoc/pyiron.vasp.structure.html#pyiron.vasp.structure.read_atoms which allows reading CONTCAR files but I was not sure if it is recommended to use those or rather ASE for the import. 

@Leimeroth Are you in contact with Yiting? In general it is easier to ask questions here on Github, this gives more people the chance to address them. Alternatively we also use https://stackoverflow.com/questions/tagged/pyiron for questions.

**Comment by Leimeroth** [comment]
Yes we had some discussion regarding the correct way to continue a MD simulation in VASP using pyiron.
From looking at the code I thought setting job.restart_file_dict["CONTCAR"] = "POSCAR" would probably be the best way.

TBH from looking trough the code and and how structures are written continuing MD runs seems more complex than it should be.
As far as I can tell velocities are pretty much never stored directly with the structure for some reason that is not clear to me and f.e. when writing a POSCAR or a LAMMPS data file it doesn't seem like velocities are used at all, so I am also not sure what happens when f.e. using the continue_with_final_structure() function.

**Comment by sudarsan-surendralal** [comment]
> @sudarsan-surendralal I guess we have https://pyiron.readthedocs.io/en/pyiron-0.2.17/apidoc/pyiron.vasp.structure.html#pyiron.vasp.structure.read_atoms which allows reading CONTCAR files but I was not sure if it is recommended to use those or rather ASE for the import.

Currently, our parser and even [ase's parser](https://databases.fysik.dtu.dk/ase/_modules/ase/io/vasp.html) does not handle predictor corrector positions.

>Yes we had some discussion regarding the correct way to continue a MD simulation in VASP using pyiron.
From looking at the code I thought setting job.restart_file_dict["CONTCAR"] = "POSCAR" would probably be the best way.

This is exactly how I do it too. It's not elegant but it works.

>TBH from looking trough the code and and how structures are written continuing MD runs seems more complex than it should be. As far as I can tell velocities are pretty much never stored directly with the structure for some reason that is not clear to me and f.e. when writing a POSCAR or a LAMMPS data file it doesn't seem like velocities are used at all, so I am also not sure what happens when f.e. using the continue_with_final_structure() function.

For LAMMPS there is an option to write the restart file (which is like the CONTCAR file) from which the velocities can be transferred to the next run. But I agree, there is currently no standard way to do this in pyiron.
